src/blocks/custom.rs

```diff
@@ -115,7 +116,8 @@ impl Block for Custom {
         let mut update = false;
 
         if let Some(ref on_click) = self.on_click {
-            Command::new("sh").args(&["-c", on_click]).output().ok();
+            Command::new(env::var("SHELL").unwrap_or_else(|_| "sh".to_owned()))
```

Comment:
As it's intended to use a literal here, simply using `unwrap_or` here might be better?

Reply:
Oh, yeah sure. Pretty new to this so I always appreciate comments ^^